pkg/network/netlink/conntracker.go

```diff
@@ -281,30 +259,6 @@ func (ctr *realConntracker) run() {
 			}
 		}
 	}()
-
-	go func() {
-		for range ctr.compactTicker.C {
-			ctr.compact()
-		}
-	}()
-}
-
-func (ctr *realConntracker) compact() {
-	ctr.Lock()
-	defer ctr.Unlock()
-
-	gen := getCurrentGeneration(generationLength, time.Now().UnixNano())
-
-	// https://github.com/golang/go/issues/20135
```

Comment:
We might need keep this functionality of periodically re-creating the whole `map` due to this issue ☝️ 
This may no longer apply to the Go version we're using, but at the very least we should double check that

Reply:
Yeah I think it is still an issue. I'll put this back on a timer.